tests/end_to_end/selenium_test.py

```diff
@@ -18,22 +15,16 @@ def test_app():
     selenium_env_file_path = find_dotenv('.env.selenium.test')
     load_dotenv(selenium_env_file_path, override=True)
 
-    trello_client = TrelloClient(TrelloConfig())
-
-    board_id = trello_client.create_board()
```

Comment:
It looks like you've avoided using your equivalent create/delete database functions - your test _if working_ should delete the item you added now, but this approach does still leave you open to a half-finished test leaving data in the db after finishing, which could affect future test runs